.github/workflows/markdown-links-check/markdown-links-check-config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "https:\/\/mvnrepository"
+    },
+    {
+      "pattern": "http:\/\/RM_hostname"
+    },
+    {
+      "pattern": "\/docs\/img\/perf-cost.png"
+    },
+    {
+      "pattern": "\/docs\/img\/ease-of-use.png"
+    },
+    {
+      "pattern": "\/docs\/get-started\/getting-started.md"
```

Comment:
I'm confused why there are so many exceptions here.  For example, why is the link to getting-started.md an exception when there are many other instances where we are linking to other .md files with a relative path?  Similarly, why are png files an issue?  Are there clues in the verbose log as to what error the link check is getting when trying to access these links?  It shouldn't be a 404, and I'm wondering if we're missing some status codes.

Reply:
thanks, add http status code 0 400 403 and remove exceptions